controller.js

```diff
@@ -53,8 +63,11 @@ function onMessagesCreated(datastore, actor, message) {
                     // Check if the webhook applies
                     if ((webhook.resource == "all")
                         || ((webhook.resource == "messages") && (webhook.event == "created"))) {
-                        // [TODO] Check filter
-                        toNotify.push(webhook);
+                        // [TODO] Check filter 
+                        if (webhook.createdBy !== message.personId) {
```

Comment:
this test should be removed IMO, as Cisco Spark does fire the webhook even though it's the webhook creator that fired the event. Then goal here will be to reproduce the same behavior in the Emulator.

Reply:
I added this check because in practice it was generating a looping behavior where a bot was posting a message, and then getting a webhook event for that message being posted, which it responded to, etc. etc.

Maybe this doesn't happen with Spark because bots don't get notifications for message created events unless they are @mentioned.   That isn't the case for one-on-one rooms though and I'm pretty certain that a bots webhook does NOT fire when it posts to a one-on-one space either.